Apps/WebClient/src/Server/Services/UserProfileService.cs

```diff
@@ -119,7 +119,7 @@ public RequestResult<UserProfile> CreateUserProfile(CreateUserRequest createProf
                 }
             }
 
-            string email = createProfileRequest.Profile.Email;
```

Comment:
Should this be nullable?  We should always have an email shouldn't we?

Reply:
We allow for the email to be empty on creating the user profile